src/routers/filing.py

```diff
@@ -46,7 +46,7 @@ async def upload_file(
 ):
     content = await file.read()
     await submission_processor.upload_to_storage(lei, submission_id, content)
-    background_tasks.add_task(submission_processor.validate_submission, lei, submission_id, content)
+    background_tasks.add_task(submission_processor.validate_submission, lei, submission_id, content, background_tasks)
```

Comment:
let's go with getting submission updated before the background_task kicks in; we don't need 2 nested bg tasks. so have the submission dao updated here to say validation in progress; then just 1 bg task that does the validation and update the submission dao when validation completes.
when we do the first submission update with the validation in progress, pass in the session already attached to the request. so
```python
repo.update_submission(the_dao, request.state.db_session)
```
then the bg task's one, call it without passing it in.

Reply:
actually, we'll deal with this in the refactor that will happen in #51; just address the submission_id comment.